src/App/App.js

```diff
@@ -0,0 +1,32 @@
+import { Header } from '../components/Header/Header';
+import { Footer } from '../components/Footer/Footer';
+import { Slider } from '../components/FeaturedSlider/FeaturedSlider';
+import { Categories } from '../components/CategoriesCarousel/CategoriesCarousel';
+import { FeaturedProducts } from '../components/FeaturedProducts/FeaturedProducts';
+import SliderData  from "../mocks/en-us/featured-banners";
+import CategoriesData  from "../mocks/en-us/product-categories";
+import FeaturedProductsData  from "../mocks/en-us/featured-products";
+
+import './App.css';
+
+function App() {
+
+  return (
+    <>
+      <div className="page-container">
+        <Header />
+        <div className="content-wrapper">
+          <h1>Featured Models</h1>
+          <Slider slides={SliderData}/>
+          <h2>Categories</h2>
```

Comment:
As a good practice in here try to add the headers with the title of the section to the component is is describing. For Example: 
`<h2>Categories</h2>` could be inside the categories component